src/API/slot.js

```diff
@@ -11,15 +11,21 @@ class SlotDataService {
         return http.simpleGet(`/queues/${queue}/slots/${id}`)
     }
 
-    reserve(queue, id, data) {
-        return http.post(`/queues/${queue}/slots/${id}/reserve?next=cancel`, data).catch((err) => {
-            const statusCode = err.response?.status
-            if (statusCode === 403) {
-                errorAlert(err.response.data.description)
-            } else {
-                err.handleGlobally()
-            }
-        })
+    reserve(queue, id, token) {
+        return http
+            .post(
+                `/queues/${queue}/slots/${id}/reserve?next=cancel`,
+                {},
+                { headers: { Authorization: `Bearer ${token}` } }
```

Comment:
Возможно в дальнейшем появятся новые виды токенов. То есть, у них будет не `Bearer`. 
Конкретный тип токена посылается вместе с токеном, в поле `token_type`. Нужно будет подредачить OttService.

Предлагаю передавать везде токен в виде пары из его типа и самого токена. Тогда здесь было бы:
```suggestion
                { headers: { Authorization: `${token.type} ${token.string}` } }
```